test/models/test_calculator.jl

```diff
@@ -26,18 +26,46 @@ elements = (:Si, :O)
 level = M.TotalDegree()
 max_level = 15
 order = 3 
+E0s = (-158.54496821, -2042.0330099956639)
```

Comment:
This is really cryptic. Can you please redefine as a Dictionary and then change internally whatever is needed to convert that Dictionary to a suitable E0 calculator during the evaluation?

Reply:
I also think we are moving to enforcing units right now. So really it should read 
```julia
E0s = Dict( :Si => -158.54496821u"eV", :O => -2042.0330099956639u"eV")
```

Reply:
The output of `model` (not calculator) that depends on `E0s` does not have units though. How about this:

- ustrip `E0s` when the model is built, and put it inside the `meta`
- In the future when the user can change the `energy_unit` when creating potential with `M.ACEPotential`, assert that the unit of the potential is the same as `E0s`. If not, pop a warning and convert the `E0s` units to the same as the calculator.

Reply:
I also put E0s (with units) into `model.mata` since the evaluation of `model` itself does not involves units. I don't want the `meta` or `ustrip` everywhere though. Will saving the `E0s` 2 times be a problem?

Reply:
for now this is fine. The ustrip too. We need to review the whole interface once the  AtomsCalculators revision is finished and the AtomsCalculatorsUtilities is ready.